sql-cli/examples/advanced/sub_dir_1/a.sql

```diff
@@ -0,0 +1 @@
+select * from {{ sub_dir_1_sub_dir_1_1_b }}
```

Comment:
This is code that will be seen by our users if they visit our GitHub. We should come up with better file names than "a.sql", "b.sql", etc.

Reply:
fixed in c6902aa